.github/CONTRIBUTING.md

```diff
@@ -52,13 +52,13 @@ npm run test:endtoend:local -- -b chrome
 
 > Note: The `--` is important, it tells npm that the remaining arguments should be interpreted by the script itself, not by npm.
 
-You can run in multiple browsers by passing multiple `-b` flags:
+You can run in several browsers by passing several `-b` flags:
 
 ```bash
 npm run test:endtoend:local -- -b chrome -b firefox
 ```
 
-When trying to get a specific integration test to pass, it's not that helpful to continuously run _all_ tests. You can use the `--suite` flag to run tests from a single `./test/endtoend` folder. For example, `--suite thumbnails` will only run the tests from `./test/endtoend/thumbnails`. Of course, it can also be combined with one or more `-b` flags.
+When trying to get a specific integration test to pass, it’s not that helpful to continuously run _all_ tests. You can use the `--suite` flag to run tests from a single `./test/endtoend` folder. For example, `--suite thumbnails` will only run the tests from `./test/endtoend/thumbnails`. It can also be joint with one or more `-b` flags.
```

Comment:
Joined? :)

Reply:
61:315-61:323  warning  Replace `combined` with `joint`       combined    retext-simplify

Reply:
I think we can ignore `combined` for `retext-simplify`. Doesn't seem like a big issue to me.

Reply:
I've rephrased it to not use combined nor joint: ``It can also be used in conjunction with one or more `-b` flags.``. wdyt?